src/helm/proxy/clients/human_task_client.py

```diff
@@ -0,0 +1,148 @@
+from abc import ABC, abstractmethod
+import random
+import threading
+from typing import Dict, List, Optional
+
+from helm.common.hierarchical_logger import hlog
+from helm.common.cache import Cache, CacheConfig
+from helm.common.human_task_request import (
+    HumanQuestionTemplate,
+    HumanTaskRequest,
+    HumanTaskRequestResult,
+    HumanTaskTemplate,
+    HumanTaskWorkerResponse,
+)
+
+from cattrs import unstructure
+
+import surge
+from surge import questions as surge_questions
+
+
+_surge_cache_lock = threading.Lock()
+
+
+class HumanTaskClient(ABC):
+    @abstractmethod
+    def get_human_task(self, request: HumanTaskRequest) -> Optional[HumanTaskRequestResult]:
+        """Get responses to a human task.
+
+        Creates the human task on a labeller platform if it does not already exist.
+        Returns HumanTaskRequestResult if worker answers are complete, or None otherwise.
+        The intended use is to call it once to create the task, wait a while, and then call it
+        later to fetch answers."""
+        pass
+
+
+class RandomHumanTaskClient(HumanTaskClient):
+    """A HumanTaskClient that returns random choices for debugging."""
+
+    def get_human_task(self, request: HumanTaskRequest) -> Optional[HumanTaskRequestResult]:
+        workers: List[HumanTaskWorkerResponse] = []
+        for _ in range(request.template.num_workers):
+            answers: List[str] = []
+            for question in request.template.questions:
+                if question.question_type != "multiple_choice":
+                    raise ValueError("Currently, only multiple_choice questions are supported")
+                answers.append(random.choice(question.options))
```

Comment:
I know this doesn't matter here, but we should just get in the habit of fixing random seeds.